### PR TITLE
Adding a new file

### PR DESCRIPTION
**Pull request by csjoin99**
# We add a new file :cry: :sudan: 
This file will be important to find the cure of the covid-19

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test
```
Comment (hectorsum) [inline test.txt:7]: # Modify this file
Diego please add more comments to this file so that I can accept the pull request.

**Review by hectorsum — COMMENTED** [review]
Modify this file, please.

File: test.txt (hunk 1 above)
Comment (csjoin99) [reply test.txt:7]: I dont want :angry:

**Review by hectorsum — APPROVED** [review]
Ok, we will merge this to the master.

**Pull request merged**